### ISSUE TITLE
Clicking fast on the same link after pressing "stay" with not trigger the alert anymore

### ISSUE THREAD

**Issue by kushtra24** [codedance/jquery.AreYouSure#136]
After I click a button or close and the alert pups-up, I press "stay" than immediately press the same button the alert does not pop-up anymore. and the browser let's me leave without any alert.

But if I wait for a second or two then the alert works again.

**Comment by Moggieggwp** [comment]
Yeah, I had the same issue, it's because there is a delay 900 milliseconds,
`window.setTimeout(function () { window.aysHasPrompted = false; }, 900);`
So you can manually change it to 0 and that's ok
